.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
     secure: RGovyUnMw3fp/bHZi058JvANT1rYmNqrsuSYew0cIgirO6YbMHr/rsjwCm1FTYpBl8s1zgr+u2b8ftYnfnCz2YT+Aip4NWrVYpVU0FEmfytGILrnUS0pjlt8m7fU9AKR1ElOSll7yw7e1kftynN39Q321etvwbLZcXon6zz0suE=
 
 matrix:
+    fast_finish: true
```

Comment:
The documentation says that it will fail the build early, not abort the other VMs. It just means it turns the attention of the developer earlier to the failure, which hopefully reaches them before they have switched the context.